app/room/__test__/page.test.tsx

```diff
@@ -0,0 +1,15 @@
+import Room from "../page";
+import React from 'react';
+import { render } from '@testing-library/react';
+
+
+describe('With React Testing Library', () => {
+
```

Comment:
instead of a sample test can you please write tests for the Header component.

Since that is part of issue #10

Reply:
Ok, no problem

Reply:
![image](https://github.com/bgwebagency/redesigner.io/assets/13310363/c148120d-9d7c-4fb9-aca6-75095b031bf1)

page.test.tsx seems to be failing. Please fix or remove this test file for now.